commands/camera/SwitchCameraCommand.java

```diff
@@ -0,0 +1,35 @@
+package frc.robot.core751.commands.camera;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.core751.subsystems.Camera;
+
+public class SwitchCameraCommand extends CommandBase {
+    private int m_device;
+    private Camera m_cameraSubsystem;
+    private boolean m_finished = false;
+
+    public SwitchCameraCommand(Camera cameraSubsystem, int initialDevice) {
+        m_device = initialDevice;
```

Comment:
in this state i believe this command only sets it to a specific camera defined once on the commands construction instead of cycling between cameras is that the desired behavior?

Reply:
This class has been updated now, so it should cycle through the cameras